### ISSUE TITLE
Various weirdness of "fixed CURRENCY VALUE ... end fixed" (BZ#789)

### ISSUE THREAD

**Issue by tbm** [ledger/ledger#789]
Note: the issue was created automatically with bugzilla2github

Bugzilla bug ID: BZ#789
From: Bradley M. Kuhn (@bkuhn)
CC:   @thdox

**Comment by tbm** [comment]
Bradley, as John pointed out in a comment, I think this is because you didn't
use the correct syntax.

With:
```
apply fixed € 0.9000

2012/07/01 First
    Accrued:ProjectB          €-1,000.00
    Expenses:ProjectB          €1,000.00

end
```
I get the expected result:
```
$ ledger -f c bal -V
                -900  Accrued:ProjectB
                 900  Expenses:ProjectB
--------------------
                   0
```

**Comment by tbm** [comment]
> Bradley, the correct syntax is "apply fixed" and "end", not "fixed" and
> "endfixed" any longer.

The docs still have "fixed" though.
